### PR TITLE
fix: backslashes in f-string error on file drop dupe widget

### LINKED ISSUE #288: [Bug]: Python complaining about backslashes inside f-string expression in dropped file dupe resolution widget
### Checklist

- [X] I am using an up-to-date version.
- [X] I have read the [documentation](https://github.com/TagStudioDev/TagStudio/blob/main/doc/index.md).
- [X] I have searched existing [issues](https://github.com/TagStudioDev/TagStudio/issues).

### TagStudio Version

e5a0e5aa9bad9ff275002549be41474b4e89c0ba

### Operating System & Version

Windows 11

### Description

```
Traceback (most recent call last):
  File "C:\Users\***\GitHub\TagStudio\TagStudio\tag_studio.py", line 9, in <module>
    from src.qt.ts_qt import QtDriver
  File "C:\Users\***\GitHub\TagStudio\TagStudio\src\qt\ts_qt.py", line 86, in <module>
    from src.qt.modals.drop_import import DropImport
  File "C:\Users\***\GitHub\TagStudio\TagStudio\src\qt\modals\drop_import.py", line 173
    )
    ^
SyntaxError: f-string expression part cannot include a backslash
```

### Expected Behavior

App should launch as expected

### Steps to Reproduce

- Clone e5a0e5aa9bad9ff275002549be41474b4e89c0ba
- Launch Python module

### Logs

See description

### PR DESCRIPTION
**Pull request by Technoguyfication**
Refactor excessively long f-string into separate variables to resolve issue

Fixes #288

### CODE REVIEW

File: tagstudio/src/qt/modals/drop_import.py [hunk 1]
```diff
@@ -167,10 +167,11 @@ def duplicates_choice(self) -> int:
         dupes_to_show = self.duplicate_files
         if len(self.duplicate_files) > display_limit:
             dupes_to_show = dupes_to_show[0:display_limit]
+        
+        dupes_str = '\n    '.join(map(lambda path: str(path),dupes_to_show))
+        dupes_more = "\nand {len(self.duplicate_files)-display_limit} more " if len(self.duplicate_files) > display_limit else '\n'
```
Comment (CyanVoxel) [inline tagstudio/src/qt/modals/drop_import.py:172]: `dupes_more`  still has f-string syntax without the f-string

Comment (Technoguyfication) [reply tagstudio/src/qt/modals/drop_import.py]: Nice catch, resolved. Looks like I forgot to test it with >5 duplicate files

File: tagstudio/src/qt/modals/drop_import.py [hunk 2]
```diff
@@ -167,10 +167,11 @@ def duplicates_choice(self) -> int:
         dupes_to_show = self.duplicate_files
         if len(self.duplicate_files) > display_limit:
             dupes_to_show = dupes_to_show[0:display_limit]
+        
+        dupes_str = '\n    '.join(map(lambda path: str(path),dupes_to_show))
+        dupes_more = f"\nand {len(self.duplicate_files)-display_limit} more " if len(self.duplicate_files) > display_limit else '\n'
```
Comment (CyanVoxel) [inline tagstudio/src/qt/modals/drop_import.py:172]: But now there's a backslash inside of the f-string again, causing the issue in #289 😅

Comment (Technoguyfication) [reply tagstudio/src/qt/modals/drop_import.py:172]: That's a backslash inside the template, not the expression. Python was upset about backslashes being between the `{ }` curly brackets. It seems to be running for me with these changes on 3.11.2

Comment (CyanVoxel) [reply tagstudio/src/qt/modals/drop_import.py:172]: Ah interesting, in that case if it works then I've got no issue with it

**Comment by CyanVoxel** [comment]
I can do the Ruff format, everything else looks good!

**Pull request merged**